core/templates/dev/head/pages/Base.ts

```diff
@@ -28,6 +28,14 @@ oppia.controller('Base', [
     $scope.currentLang = 'en';
     $scope.iframed = UrlService.isIframed();
     $scope.siteFeedbackFormUrl = SITE_FEEDBACK_FORM_URL;
+    $scope.pageUrl = UrlService.getCurrentLocation().href;
+    $scope.getAssetUrl = function(path) {
+      if (DEV_MODE) {
+        return UrlService.getOrigin() + path;
+      } else {
+        return UrlService.getOrigin() + '/build' + path;
+      }
+    };
```

Comment:
Can you move this into _UrlInterpolationService.ts_?

Reply:
Okay